webcompat/static/css/development/components/navbar.css

```diff
@@ -68,6 +68,11 @@
     vertical-align: middle;
   }
 
+  /* .is-hidden class added when add-on is installed */
+  .wc-Navbar .is-hidden {
+    display: none;
+  }
+
```

Comment:
Where class will be add?  all links https://github.com/webcompat/webcompat.com/blob/master/webcompat/templates/topbar.html#L8 ?

something like `<a class="wc-Navbar-link is-hidden js-Navbar-link"` ?